projects/addon-table/components/table-pagination/table-pagination.component.ts

```diff
@@ -38,6 +38,9 @@ export class TuiTablePaginationComponent {
     @Input()
     public size = this.options.size;
 
+    @Input()
+    public showFirstLastPageButtons = false;
```

Comment:
Sorry for taking so long to review this. We need shorter names and we do not need to deprecate and rename things, let's do this with minimum intervention to how it works right now. If we only want it in 4.0, we can just rename everything to better suited names.